core/plannernotes.c

```diff
@@ -272,21 +272,21 @@ void add_plan_to_notes(struct diveplan *diveplan, struct dive *dive, bool show_d
 					if (dp->setpoint) {
 						snprintf(temp, sz_temp, translate("gettextFromC", "(SP = %.1fbar)"), (double) nextdp->setpoint / 1000.0);
 						len += snprintf(buffer + len, sz_buffer - len, "<td style='padding-left: 10px; color: red; float: left;'><b>%s %s</b></td>", gasname(&newgasmix),
-									temp);
+							temp);
```

Comment:
probably better to either move `temp` on the previous line or move more text on this new line.